Example/src/LightboxExample.js

```diff
@@ -0,0 +1,259 @@
+
+  const backdropOpacity = useSharedValue(0);
+  const scale = useSharedValue(1);
+
+  const {
+    x,
+    item,
+    width,
+    height,
+    targetWidth,
+    targetHeight,
+    sv: imageOpacity,
+  } = activeImage;
+
+  const y = activeImage.y - Header.HEIGHT;
+
+  const target = useSharedVector(
+    0,
+    (dimensions.height - targetHeight) / 2 - Header.HEIGHT
+  );
+
+  const translate = useSharedVector(0, 0);
+
+  const onPan = useAnimatedGestureHandler({
+    onActive: event => {
+      translate.x.value = event.translationX;
+      translate.y.value = event.translationY;
+
+      scale.value = interpolate(
+        translate.y.value,
+        [-200, 0, 200],
+        [0.65, 1, 0.65],
+        Extrapolate.CLAMP
+      );
+
+      backdropOpacity.value = interpolate(
+        translate.y.value,
+        [-100, 0, 100],
+        [0, 1, 0],
+        Extrapolate.CLAMP
+      );
+    },
+
+    onEnd: (event, ctx) => {
+      if (Math.abs(translate.y.value) > 40) {
+        target.x.value = translate.x.value - target.x.value * -1;
+        target.y.value = translate.y.value - target.y.value * -1;
+
+        translate.x.value = 0;
+        translate.y.value = 0;
+
+        animationProgress.value = withTiming(0, timingConfig, () => {
+          imageOpacity.value = 1;
+          onClose();
+        });
+
+        backdropOpacity.value = withTiming(0, timingConfig);
+      } else {
+        backdropOpacity.value = withTiming(1, timingConfig);
+        translate.x.value = withTiming(0, timingConfig);
+        translate.y.value = withTiming(0, timingConfig);
+      }
+
+      scale.value = withTiming(1, timingConfig);
+    },
+  });
+
+  const imageStyles = useAnimatedStyle(() => {
+    const interpolateProgress = range =>
+      interpolate(animationProgress.value, [0, 1], range, Extrapolate.CLAMP);
+
+    const translateY =
+      translate.y.value + interpolateProgress([y, target.y.value]);
+    const translateX =
+      translate.x.value + interpolateProgress([x, target.x.value]);
+
+    return {
+      position: 'absolute',
+      top: translateY,
+      left: translateX,
+      width: interpolateProgress([width, targetWidth]),
+      height: interpolateProgress([height, targetHeight]),
+      transform: [
+        {
+          scale: scale.value,
+        },
+      ],
+    };
+  });
+
+  const backdropStyles = useAnimatedStyle(() => {
+    return {
+      opacity: backdropOpacity.value,
+    };
+  });
+
+  runOnUI(() => {
+    'worklet';
+
+    animationProgress.value = withTiming(1, timingConfig, () => {
+      imageOpacity.value = 0;
+    });
+    backdropOpacity.value = withTiming(1, timingConfig);
+  })();
+
+  return (
+    <View style={StyleSheet.absoluteFillObject}>
+      <Animated.View style={[styles.backdrop, backdropStyles]} />
+
+      <PanGestureHandler onGestureEvent={onPan}>
+        <Animated.View style={StyleSheet.absoluteFillObject}>
+          <AnimatedImage source={{ uri: item.uri }} style={imageStyles} />
```

Comment:
it could be done using object destructing like `const { uri } = item` and then `<AnimatedImage source={{ uri }} style={imageStyles} />`

Reply:
It required another destructuring of the item which is destructured too. But yeah, changed it.